src/contracts/test/GPv2SettlementTestInterface.sol

```diff
@@ -1,10 +1,17 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
 pragma solidity ^0.6.12;
-pragma experimental ABIEncoderV2;
 
 import "../GPv2Settlement.sol";
 
 contract GPv2SettlementTestInterface is GPv2Settlement {
+    constructor(GPv2SimpleAuthentication _controller)
+        public
+        GPv2Settlement(_controller)
+    // solhint-disable-next-line no-empty-blocks
+    {
+
+    }
```

Comment:
The auto-formatter is funny sometimes :see_no_evil:

Reply:
Yup, this is why I took the opportunity to include the comment that was previously there.